changelog/README.md

```diff
@@ -29,6 +29,7 @@
 - [improvement] JAVA-1772: Revisit multi-response callbacks
 - [new feature] JAVA-1537: Add remaining socket options
 - [bug] JAVA-1756: Propagate custom payload when preparing a statement
+- [new feature] JAVA-1832: Add EC2MultiRegionAddressTranslator
```

Comment:
Nit: we put new entries first in the 4.x changelog

Reply:
d'oh, will fix